osbs/conf.py

```diff
@@ -554,3 +543,36 @@ def get_platform_node_selector(self, platform):
                 nodeselector = dict([k.strip(), v.strip()] for (k, v) in raw_nodeselector.items())
 
         return nodeselector
+
+    def get_scratch_build_node_selector(self):
+        nodeselector = {}
+        nodeselector_str = self._get_value("scratch_build_node_selector", self.conf_section,
+                                           "scratch_build_node_selector")
+        if nodeselector_str and nodeselector_str != 'none':
```

Comment:
yeah as many other methods in conf.py, so I thought is better keep it nice and separate